app/decorators/user_decorator.rb

```diff
@@ -116,9 +116,9 @@ def should_acknowledge_personal_key?(session)
   end
 
   def recent_events
-    events = user.events.order('updated_at DESC').limit(MAX_RECENT_EVENTS).map(&:decorate)
+    events = user.events.order('created_at DESC').limit(MAX_RECENT_EVENTS).map(&:decorate)
     identities = user.identities.order('last_authenticated_at DESC').map(&:decorate)
-    (events + identities).sort { |thing_a, thing_b| thing_b.happened_at <=> thing_a.happened_at }
+    (events + identities).sort_by(&:happened_at).reverse
```

Comment:
I read some [documentation](https://apidock.com/ruby/Enumerable/sort_by) and an [SO post](https://stackoverflow.com/a/2651028) which suggest `sort_by` is faster than `sort`. Even if they are close in performance, `sort_by` has a readability advantage. Not sure how this reconciles with [this comment I saw in `.fasterer.yml`](https://github.com/18F/identity-idp/blob/9939092773063f3edab05698a514f7f77b367dc2/.fasterer.yml#L5).